data_structures/src/chain.rs

```diff
@@ -537,6 +540,30 @@ pub struct BlockMerkleRoots {
     pub tally_hash_merkle_root: Hash,
 }
 
+/// Block merkle tree roots
+#[derive(Debug, Eq, PartialEq, Clone, Serialize, Deserialize, ProtobufConvert, Default)]
+#[protobuf_convert(pb = "witnet::SuperBlock")]
+pub struct SuperBlock {
+    /// Merkle root of the last superblock period blocks data request merkles root
```

Comment:
The description is not super clear to me, maybe something like
"Merkle root of the data requests in the blocks created since the last Superblock" ?